python/plugins/processing/algs/saga/SagaAlgorithmProvider.py

```diff
@@ -91,34 +81,28 @@ def _loadAlgorithms(self):
             ProcessingLog.addToLog(ProcessingLog.LOG_ERROR,
                                    self.tr('Problem with SAGA installation: SAGA was not found or is not correctly installed'))
             return
-        if version not in self.supportedVersions:
-            lastVersion = sorted(self.supportedVersions.keys())[-1]
-            if version > lastVersion:
-                version = lastVersion
-            else:
-                ProcessingLog.addToLog(ProcessingLog.LOG_ERROR,
-                                       self.tr('Problem with SAGA installation: installed SAGA version (%s) is not supported' % version))
-                return
+
+        if version not in ['2.3.0', '2.3.1']:
```

Comment:
@ghtmtt I compiled it without gui. and this from terminal:
`saga_cmd --version`

reports:
`SAGA Version: 2.3.2`